var/spack/repos/builtin/packages/codar-cheetah/package.py

```diff
@@ -6,20 +6,19 @@
 from spack import *
 
 
-class CodarCheetah(Package):
+class CodarCheetah(PythonPackage):
     """CODAR Cheetah:
     The CODAR Experiment Harness for Exascale science applications.
     """
 
+    maintainers = ['kshitij-v-mehta']
+
     homepage = "https://github.com/CODARcode/cheetah"
     url      = "https://github.com/CODARcode/cheetah/archive/v0.1.tar.gz"
     git      = "https://github.com/CODARcode/cheetah.git"
 
-    version('develop', branch='master')
+    version('develop', branch='dev')
+    version('0.5', sha256='f37a554741eff4bb8407a68f799dd042dfc4df525e84896cad70fccbd6aca6ee')
     version('0.1', sha256='281564f8ae57a70ce28457616fde26247ea4efb29e55c7bf89a782a259a1a028')
```

Comment:
It looks like 0.1 didn't use the `PythonPackage` build system. Should we drop this version?

Reply:
Ok, I have updated the package